positioning1/sandbox.css

```
@@ -0,0 +1,35 @@
+*,
+body {
+    margin: 0;
+    padding: 0;
+}
+
+div {
+    width: 200px;
+    height: 200px;
+}
+
+.yellow {
+    background-color: yellow;
+    position: fixed;
+    /* pentru a pozitiona patratul relativ la toata pagina */
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    /*mijlocul patratului galben in mijlocul paginii */
+}
+
+.blue {
+    background-color: blue;
+}
+
+.green {
+    background-color: green;
+}
+
+.blue,
+.green {
+    position: absolute;
+    top: 100px;
+    left: 100px;
+}
```

Este ok cum ai facut, dar poti simplifica si mai tare lucrurile si nu trebuie sa pui px. Daca pui px si in viitor poate vine cineva si vrea sa schimbe dimensiunile div-ului, atunci si aici trebuie recalculat.
O sugestie ar fi, pentru ca ai pus fiecare din div-uri unul in interiorul celuilalt:

```suggestion
.blue,
.green {
    position: absolute;
    top: 50%;
    left: 50%;
}
```
Pentru ca fiecare div va fi positionat in functie de pozitia parintelui, punand 50% o sa fii 100% sigura ca indiferent de dimensiunea parintelui, coltul din stanga al urmatorului patrat o sa fie in mijloc;